modules/udparsers/m2c00/spcn_trace.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python3
```

Comment:
I wonder if this should be in a separate module/path...  It could then just call into the udparsers modules as appropriate.  What would you think?

Reply:
Yeah, that makes sense.  Do you have a suggestion?  modules/scripts?  modules/io_drawer?

Reply:
If that should go in modules/io_drawer, maybe most of the others should too with just the true error log plugin m2c00.py in the udparsers path?

Reply:
Is this only used by the `m2c00` module? I thought that was the whole point of putting all of the user data parsers in subdirectories like `m2c00/m2c00.py` so that any supporting modules could be in that same directory.

Reply:
Zane, I agree that other modules that are unique to an error log plugin should go in the plugin directory.

In this case, most of the modules are common across two different use cases:
* Error log plugin
* Standalone tool for parsing SPCN Trace (IO Drawer Dump) data.

Matt and I talked, and we decided to move the common code into a separate module/path.  The plugin directory will still be used for modules specify to plugin functionality.

Reply:
Ok, sounds good.